src/cmd/linuxkit/run_qemu.go

```diff
@@ -80,19 +80,59 @@ func runQemu(args []string) {
 	remArgs := flags.Args()
 
 	if len(remArgs) == 0 {
-		fmt.Println("Please specify the prefix to the image to boot")
+		fmt.Println("Please specify the path to the image to boot")
 		flags.Usage()
 		os.Exit(1)
 	}
-	prefix := remArgs[0]
+	path := remArgs[0]
 
-	// Print warning if conflicting UEFI and ISO flags are set
-	if *uefiBoot && *isoBoot {
-		log.Warnf("Both -iso and -uefi have been used")
+	// if user specified a path that exists and ends in .iso they meant an ISO
```

Comment:
I think it would be overkill to start prodding around inside the images we boot.